RecoLocalMuon/RPCRecHit/cscObjectMapESProducer.py

```diff
@@ -0,0 +1,3 @@
+import FWCore.ParameterSet.Config as cms
+
+cscObjectMapESProducer = cms.EDProducer( 'CSCObjectMapESProducer' )
```

Comment:
Ops, that should not be there at all...

By the way, I wrote it as an EDProducer because edmPythonConfigToCppValidation does not like ESProducers...